apps/public-docsite-v9/src/Concepts/BrowserSupportMatrix.stories.mdx

```diff
@@ -23,6 +23,35 @@ FluentUI will not be providing polyfills for features that we expect our browser
 use/support of older browsers that don't support certain browser, ES, or CSS features (i.e. `ResizeObserver`),
```

Comment:
I think the paragraph on polyfills should read something more like this (can't do a suggestion due to the lines not being changed).

"FluentUI will not be providing our own polyfills for features that we expect our browser support matrix to already support. There are often well-tested third-party polyfills available that applications can use to fill in the gaps. Consumers who require the use/support of older browsers that don't support certain browser, ES, or CSS features (i.e. `ResizeObserver`), will need to implement or use a third-party polyfill to provide the functionality. See the partial support section below for features that may need to be polyfilled"